todo/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Todo from './components/Todo';
+import {Route} from 'react-router-dom';
+import Form from './components/Form';
+
+function App() {
+  return (
+    <div className="App">
+      <Route exact path="/" render={(props) =>  <Todo {...props} />} />
+      <Route render={() => <Form />} />
+    </div>
```

Comment:
Line 12 - if you don't intend to have a component render based on a path, then you can just take it out of the `<Route />` tag.